### ISSUE TITLE
Globally change defaults

### ISSUE THREAD

**Issue by Kyle-Thompson** [neovim/nvim-lsp#154]
Is there a way to change defaults for all servers in one place? For example, to get this working with ncm2 you need to add `on_init = ncm2.register_lsp_source;`. Given that I'll want this option added every time, is there a place to override the global defaults and include it? It should still be possible to override it again later for one particular server if a need arises, it would just be nice to reduce duplicate code.

**Comment by h-michael** [comment]
Would do you try this [PR](https://github.com/neovim/nvim-lsp/pull/149)?

```vim
lua << EOF
	local nvim_lsp = require"nvim_lsp"
	nvim_lsp.util.default_config = vim.tbl_extend(
		"force",
		nvim_lsp.util.default_config,
		{on_init = ncm2.register_lsp_source})
	nvim_lsp.some_ls.setup{}
<< EOF
```

**Comment by h-michael** [comment]
resolved #149
